tests/e2e/api-core-tests/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
This package should be added to the root package similar to https://github.com/woocommerce/woocommerce/blob/trunk/package.json#L50:L53.

Also, a script in the root package would be handy.

Reply:
Ah nice catch. Added that into the root package and the exclude array in `subsets`. I also added in a script to run the API tests like we have for `test:e2e`. Right now this just `cd`'s into the API test directory and runs the `jest --group=api` command. I wasn't sure if we needed a full script here (like `wc-e2e`) but let me know your thoughts, and if you think we should include any other commands. Thanks!

Reply:
That script is okay for the initial PR. Before publishing the package we should add a full script implementation like `wc-e2e`.

Reply:
Sounds good, I added an issue for that here: https://github.com/woocommerce/woocommerce/issues/30656